include/drivers/yen/withPoints_ksp_driver.h

```diff
@@ -50,18 +51,17 @@ typedef struct Path_rt Path_rt;
 extern "C" {
 #endif
 
-    //  CREATE OR REPLACE FUNCTION pgr_withPointKsp(
-    //  edges_sql TEXT,
-    //  points_sql TEXT,
-    //  start_pid BIGINT,
-    //  end_pid BIGINT,
-    //  directed BOOLEAN DEFAULT true,
     int do_pgr_withPointsKsp(
-            Edge_t  *edges,             size_t total_edges,
-            Point_on_edge_t  *points,       size_t total_points,
-            Edge_t  *edges_of_points,   size_t total_edges_of_points,
+            Edge_t  *edges,           size_t total_edges,
```

Comment:
Now that you are touching a driver header file, we are no using parameters names any more on headers when possible.
In this one is possible, for example:
https://github.com/pgRouting/GSoC-pgRouting/blob/main/include/drivers/dijkstra/dijkstra_driver.h#L57